### PR TITLE
i18n(fr): Add `guides/cms/storyblok` in French language

### PR DESCRIPTION
**Pull request by thomasbnt**
#### Description (required)
Add `guides/cms/storyblok` in French language

#### Related issues & labels (optional)
i18n label.

### CODE REVIEW

File: src/content/docs/fr/guides/cms/storyblok.mdx [hunk 1]
```diff
@@ -0,0 +1,543 @@
...
+
+## Créer un blog avec Astro et Storyblok
+
+Avec l'intégration mise en place, vous pouvez maintenant créer un blog avec Astro et Storyblok.
+
+### Prérequis
+
+1. **Un espace Storyblok** - Pour ce tutoriel, nous recommandons d'utiliser un nouvel espace. Si vous avez déjà un espace avec des Bloks, n'hésitez pas à les utiliser, mais vous devrez modifier le code pour qu'il corresponde aux noms et aux types de contenu des Bloks.
+
+2. **Un projet Astro intégré à Storyblok** - Voir [integrating with Astro](#intégration-avec-astro) pour les instructions sur la façon de mettre en place l'intégration.
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/cms/storyblok.mdx:227]:
```suggestion
2. **Un projet Astro intégré à Storyblok** - Voir [les intégrations avec Astro](#intégration-avec-astro) pour les instructions sur la façon de mettre en place l'intégration.
```

File: src/content/docs/fr/guides/cms/storyblok.mdx [hunk 2]
```diff
@@ -0,0 +1,543 @@
...
+    <StoryblokComponent blok={content} />
+  </body>
+</html>
+```
+
+Pour générer des pages pour tous les articles de votre blog, créez une page `.astro` qui créera des routes dynamiques. Cette approche varie selon que vous utilisez la **génération de site statique** (par défaut) ou le **rendu côté serveur**.
+
+#### Génération de sites statiques
+
+Si vous utilisez la génération de site statique par défaut d'Astro, vous utiliserez [dynamic routes](/fr/core-concepts/routing/#routes-dynamiques) et la fonction `getStaticPaths` pour générer les pages de votre projet.
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/cms/storyblok.mdx:401]:
```suggestion
Si vous utilisez la génération de site statique par défaut d'Astro, vous utiliserez les [routes dynamiques](/fr/core-concepts/routing/#routes-dynamiques) et la fonction `getStaticPaths` pour générer les pages de votre projet.
```

File: src/content/docs/fr/guides/cms/storyblok.mdx [hunk 3]
```diff
@@ -0,0 +1,543 @@
...
+
+##### Ajouter un webhook à Storyblok
+
+Dans votre espace Storyblok **Settings**, cliquez sur l'onglet **Webhooks**. Collez l'URL du webhook que vous avez copié dans le champ **Story published & unpublished** et cliquez sur <kbd>Save</kbd> pour créer un webhook.
+
+Désormais, chaque fois que vous publierez un nouvel article, une nouvelle construction sera déclenchée et votre blog sera mis à jour.
+
+## Ressources officielles
+
+- Storyblok fournit une [Astro Intégration](https://www.storyblok.com/mp/announcing-storyblok-astro) pour ajouter Storyblok à votre projet.
```
Comment (McFlyPartages) [inline src/content/docs/fr/guides/cms/storyblok.mdx:537]:
```suggestion
- Storyblok fournit une [Intégration Astro](https://www.storyblok.com/mp/announcing-storyblok-astro) pour ajouter Storyblok à votre projet.
```

**Review by McFlyPartages — APPROVED** [review]
Petites suggestions, mais pas obligatoires.

Par contre, il faut choisir soit histoire ou story lol

Good Job LGTM

**Review by Yan-Thomas — APPROVED** [review]
LGTM! Thanks peeps 🚀

**Pull request merged**